src/main/java/at/ac/tuwien/kr/alpha/common/predicates/ExternalFixedEvaluable.java

```diff
@@ -0,0 +1,81 @@
+package at.ac.tuwien.kr.alpha.common.predicates;
+
+import at.ac.tuwien.kr.alpha.common.terms.ConstantTerm;
+import at.ac.tuwien.kr.alpha.common.terms.Term;
+import org.apache.commons.lang3.ClassUtils;
+
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+import java.util.List;
+import java.util.Set;
+
+public class ExternalFixedEvaluable implements Predicate, Evaluable {
```

Comment:
Maybe call this `ExternalPredicate` ?

Reply:
This would clash with `ExternalNativePredicate`, now called `ExternalPredicate` (according to your suggestion). So I went for `ExternalMethodPredicate`.